src/Layer/PointCloudLayer.js

```diff
@@ -90,6 +90,18 @@ function changeIntensityRange(layer) {
     }
 }
 
+function changeElevationRange(layer) {
+    if (layer.material.elevationRange) {
+        layer.material.elevationRange.set(layer.minElevationRange, layer.maxElevationRange);
+    }
+}
+
+function changeAngleRange(layer) {
+    if (layer.material.angleRange) {
+        layer.material.angleRange.set(layer.minAngleRange, layer.maxAngleRange);
+    }
```

Comment:
- The properties `minAngleRange` and `maxAngleRange` shall be documented.
- Add a guard when `minAngleRange` == `maxAngleRange` to prevent divide by zero
- Use the `?.` operator:
```suggestion
    layer.material.angleRange?.set(layer.minAngleRange, layer.maxAngleRange);
```